registration/validators.py

```diff
@@ -69,3 +70,12 @@ def validate_client_interaction(interaction):
 def validate_interactions(interactions):
     for interaction in interactions.values():
         validate_client_interaction(interaction)
+
+
+def validate_field_order(field_order, role):
+    Field = apps.get_model("registration", "Field")
+    if (
+        not field_order
+        > Field.objects.filter(role=role).aggregate(Max("order"))["order__max"]
```

Comment:
i think previously this had to be exactly the max + 1, is there a reason we changed this to a general greater than?

Reply:
tbh I thought I had a reason regarding the soft deletion of fields, but now that I think about it... there isn't a reason